Source/Blazorise/Base/BaseInputComponent.razor.cs

```diff
@@ -62,7 +62,7 @@ public override async Task SetParametersAsync( ParameterView parameters )
                     }
                     else
                     {
-                        ExecuteAfterRender( () => FocusAsync() );
+                        ExecuteAfterRender( () => Focus() );
```

Comment:
Does this ExecuteAfterRender support async?

Reply:
It does.